charts/logging-apps/Chart.yaml

```
@@ -17,82 +17,7 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
```

given the CVE, can you add this?
```suggestion
  artifacthub.io/containsSecurityUpdates: "true"
  artifacthub.io/changes: |
```

---

Ah, i didn't know that existed. Added.